app/scripts/services/authorization.js

```diff
@@ -96,16 +96,22 @@ angular.module("openshiftConsole")
       return _.get(cachedRulesByProject.get(projectName || currentProject), ['rules']);
     };
 
+    var checkRulesForResourceVerb = function(rules, resource, verb) {
+      return _.contains(rules[resource], verb) || _.contains(rules[resource], '*');
+    };
+
     var canI = function(verb, resource, projectName) {
```

Comment:
it's part of the rules, but I don't see it being provided to `canI` separately, which means it's hard to check if  the literal API group provided to `canI` *or* `*` is allowed without parsing the provided resource. Also, if you're using `/` to delimit the api group, there's no way to distinguish between an api group and a subresource